docs/handshake.md

```diff
@@ -841,312 +1190,337 @@ message when the sender seems to linger with retransmission).
 
 (5) `SRT_OPT_REXMITFLG`: The party uses the REXMIT flag.
 
-This flag should be always set. It's a **Special legacy compatibility flag**,
-see below for more details.
+This **special legacy compatibility flag** should be always set. See below 
+for more details.
 
-(6) `SRT_OPT_STREAM`   : The party uses stream type transmission
+(6) `SRT_OPT_STREAM`: The party uses stream type transmission.
 
 This is introduced in HSv5 only. When set, the party is using a stream
 type transmission (file transmission with no boundaries). In HSv4 this
-flag does not exist and therefore it's always clear, which corresponds
-with the fact that HSv4 supports the Live mode only.
+flag does not exist, and therefore it's always clear, which corresponds
+to the fact that HSv4 supports Live mode only.
 
-**Special legacy compatibility flags**
+**Special Legacy Compatibility Flags**
 
-`SRT_OPT_HAICRYPT` and `SRT_OPT_REXMITFLG` define special cases of
-interpretation of the contents in the SRT header for payload packets.
+The `SRT_OPT_HAICRYPT` and `SRT_OPT_REXMITFLG` fields define special cases for 
+the interpretation of the contents in the SRT header for payload packets.
 
 The SRT header contains an unusual field designated as `PH_MSGNO`,
 which contains first some extra flags that occupy the most significant
-bits in this field, and the rest is assigned to the Message Number.
+bits in this field (the rest are assigned to the Message Number).
 Some of these extra flags were already in UDT, but SRT added some
-more by stealing more bits from the Message Number subfield:
+more by stealing bits from the Message Number subfield:
 
-1. Encryption Key flags (2 bits). Controlled by `SRT_OPT_HAICRYPT`,
-this field contains a value that declares as to whether the payload is
+1. **Encryption Key** flags (2 bits). Controlled by `SRT_OPT_HAICRYPT`,
+this field contains a value that declares whether the payload is
 encrypted and with which key.
 
-2. Retransmission flag (1 bit). Controlled by `SRT_OPT_REXMITFLG`,
-this flag is clear when the payload was sent originally and set,
-when the payload was retransmitted.
+2. **Retransmission** flag (1 bit). Controlled by `SRT_OPT_REXMITFLG`, this flag 
+is 0 when a packet is sent the first time, and 1 when it is retransmitted (i.e. 
+requested in a loss report). When the incoming packet is belated (one with a
+sequence number older than the newest received so far), this flag allows the
+Receiver to distinguish between a retransmitted packet and reordered packet.
+This is used by the "reorder tolerance" feature described in the API
+documentation under "SRTO_LOSSMAXTTL").
```

Comment:
documentation under "SRTO_LOSSMAXTTL".

Reply:
I fixed it also to:

documentation under `SRTO_LOSSMAXTTL` socket option.